spring/src/main/java/uk/ac/bbsrc/tgac/miso/spring/ajax/ExperimentControllerHelperService.java

```diff
@@ -246,33 +237,35 @@ public JSONObject getLibraryKitDescriptors(HttpSession session, JSONObject json)
   }
 
   public JSONObject addLibraryKit(HttpSession session, JSONObject json) {
-    try {
-      if (json.has("experimentId")) {
-        String experimentId = json.getString("experimentId");
-        String kitDescriptor = json.getString("kitDescriptor");
-        String lotNumber = json.getString("lotNumber");
-
-        LibraryKit lk = new LibraryKit();
-        KitDescriptor kd = requestManager.getKitDescriptorById(new Long(kitDescriptor));
-        lk.setKitDescriptor(kd);
-        lk.setLotNumber(lotNumber);
-        if (!json.has("kitDate") || isStringEmptyOrNull(json.getString("kitDate"))) {
-          lk.setKitDate(new Date());
-        }
-
-        Experiment e = requestManager.getExperimentById(new Long(experimentId));
-        e.addKit(lk);
-        e.setLastModifier(securityManager.getUserByLoginName(SecurityContextHolder.getContext().getAuthentication().getName()));
-        requestManager.saveExperiment(e);
-        Integer newStock = kd.getStockLevel() - 1;
-        kd.setStockLevel(newStock);
-        requestManager.saveKitDescriptor(kd);
-      }
-      return JSONUtils.SimpleJSONResponse("Saved kit!");
-    } catch (IOException e) {
-      log.error("add library kit", e);
-      return JSONUtils.SimpleJSONError("Failed to save library kit");
-    }
+    // TODO CS : Implement
```

Comment:
Will these non-implemented methods be implemented in the front-end stock control PR?